tests/test_core/test_new_moltype.py

```diff
@@ -192,3 +192,20 @@ def test_get_degenerate_positions(data_type, moltype):
     got = moltype.get_degenerate_positions(seq)
     expect = numpy.array([], dtype=bool)
     assert numpy.array_equal(got, expect)
+
+
+@pytest.mark.parametrize(
+    "moltype",
+    (
+        new_moltype.ASCII,
+        new_moltype.DNA,
+        new_moltype.RNA,
+        new_moltype.PROTEIN,
+        new_moltype.PROTEIN_WITH_STOP,
+    ),
+)
+def test_gaps(moltype):
```

Comment:
BYTES will fail this since it defines gap and missing as None, so implement the test that BYTES.gaps returns None or perhaps an empty set?

Reply:
Thanks, will work on next PR